android/app/src/main/java/net/minetest/minetest/MainActivity.java

```diff
@@ -114,6 +126,30 @@ private void startNative() {
 		startActivity(intent);
 	}
 
+	@RequiresApi(Build.VERSION_CODES.O)
+	private void createNotificationChannel() {
+		NotificationManager notifyManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+		if (notifyManager == null)
+			return;
+
+		String channelName = "General notification";
+		String channelDescription = "Notifications from Minetest";
```

Comment:
Can these strings be made translatable?

Reply:
Do we want Minetest to react to [ACTION_LOCALE_CHANGED](https://developer.android.com/reference/android/content/Intent#ACTION_LOCALE_CHANGED) intents (most apps) or should the notification channel information be changed when the app started (simpler)?

Reply:
> Do we want Minetest to react to [ACTION_LOCALE_CHANGED](https://developer.android.com/reference/android/content/Intent#ACTION_LOCALE_CHANGED) intents (most apps) or should the notification channel information be changed when the app started (simpler)?

Both approaches are fine by me.

Reply:
I decided to just use a simple `getString()`. The ability to react to the intent can be done in another time.